nft/nfts.yaml

```diff
@@ -300,7 +300,7 @@ paths:
       parameters:
         - $ref: '#/components/schemas/apiKey'
         - $ref: '#/components/schemas/contractAddress'
-          required: true
```

Comment:
Are we not having `contractAddress` as a required param anymore?

Feel like this is required for `getNFTsForCollection`

Reply:
now either collection slug or contract address can be passed. at least one of them is required. not sure how to express that in readme though

Reply:
Ah got it, yeah, in that case, we just have to make both optional. Though, we can call this out in the description that one of them is required.